website/blog/2020-06-16-alluxio-datacaching.md

```diff
@@ -0,0 +1,243 @@
+---
+title: Improving Presto Latencies with Alluxio Data Caching
+author: Rohit Jain
+authorURL: https://www.linkedin.com/in/jain-rohit/
+---
+
+**Facebook:** Rohit Jain, James Sun, Ke Wang, Shixuan Fan, Biswapesh Chattopadhyay, Baldeep Hira
+
+**Alluxio:** Bin Fan, Calvin Jia, Haoyuan Li
+
+
+# Introduction
+
+The Facebook Presto team has been collaborating with [Alluxio](https://www.alluxio.io/) on an open source data caching solution for Presto. 
+This is required for multiple Facebook use-cases to improve query latency for queries that scan data from remote sources, such as HDFS. 
+We have observed significant improvements in query latencies and IO scans in our experiments. 
+<br><br>
+We found Alluxio data caching to be useful for multiple use-cases in the Facebook environment. 
+For one of the Facebook internal use cases, we observed query latencies improved by **33%** (P50), **54%** (P75), and **48%** (P95). 
+We also recorded **57%** improvement in IO for remote data source scans.
+
+# Presto Architecture
+
+Presto architecture is designed to allow data and computation to scale independently. Presto workers are responsible for executing query plan fragments 
+on the data scanned from the independent and typically remote data sources. For remote data sources, Presto workers do not store any data locally which 
+enables the computation to grow elastically.
+<br><br>
+The architecture diagram below highlights the data read paths from remote HDFS source. 
+Each worker independently reads data from the remote data source. In this blog, we will be only talking about optimizations done in the read 
+operations from the remote data source.
+<br><br>
+
+
+![](/img/blog/2020-06-16-alluxio-datacaching/presto_worker.jpg)
+
+<br>
+
+This architecture allows storage and computation to scale independently. However, scanning data from remote storage can be a costly operation, 
```

Comment:
Don't bury this at the end, this is the lead topic for this section. The rest of the section is explaining why the previous architecture had this problem.

Reply:
Would you prefer it in bold?